MatrixSDKTests/MXAggregatedEditsTests.m

```diff
@@ -165,8 +222,13 @@ - (void)testEditsFromInitialSync
 
             [mxSession start:^{
 
+                MXEvent *editedEvent = [mxSession.store eventWithEventId:eventId inRoom:room.roomId];
+                
                 // -> Data from aggregations must be right
-                XCTFail(@"TODO: Write the test once we know what we should receive");
+                XCTAssertNotNil(editedEvent);
+                XCTAssertTrue(editedEvent.contentHasBeenEdited);
+                XCTAssertEqualObjects(editedEvent.unsignedData.relations.replace.eventId, editEventId);
+                XCTAssertEqualObjects(editedEvent.content[@"body"], kEditedMessageText);
```

Comment:
Can we compare `editedEvent` with the same `editedEvent` but got before doing the initial sync?
We could compare their JSONDictionary.
That will validate that the SDK updates edited event like the server.